### PR TITLE
Removes sleeps and adds Actions

### PR DESCRIPTION
**Pull request by KristinaKosko**

### CODE REVIEW

File: Framework/Framework/UI/Pages/CheapFlightPage.cs [hunk 1]
```diff
@@ -42,20 +43,20 @@ public void ClickContinue()
 
         public void SelectPrice()
         {
-            Wait.Until(ExpectedConditions.ElementIsVisible(By.ClassName("core-btn-primary fare-select")));
-            SelectRaceBtn.Click();
+            Actions clickToLocation = new Actions(DriverFactory.Driver);
```
Comment (OlegBaslak) [inline Framework/Framework/UI/Pages/CheapFlightPage.cs]: var

File: Framework/Framework/UI/Pages/CheapFlightPage.cs [hunk 2]
```diff
@@ -42,20 +43,20 @@ public void ClickContinue()
...
+            clickToLocation.MoveToElement(SelectRaceBtn[0]).Click(SelectRaceBtn[0]).Release();
         }
 
         public void SelectPossibleRace()
         {
-            Wait.Until(ExpectedConditions.ElementIsVisible(By.ClassName("flight-header__min-price")));
+            Wait.Until(ExpectedConditions.ElementIsVisible(By.XPath("//div[@class='flight-header__min-price']")));
```
Comment (OlegBaslak) [inline Framework/Framework/UI/Pages/CheapFlightPage.cs]: use Wait in DriverFactory instead

File: Framework/Framework/UI/Pages/CheapFlightPage.cs [hunk 3]
```diff
@@ -42,20 +43,20 @@ public void ClickContinue()
...
             ChooseRaceBtn.Click();
         }
 
         public void SaveTrip()
         {
-            Wait.Until(ExpectedConditions.ElementIsVisible(By.XPath("//button[@class='core-btn save-btn core-btn-wrap']")));
-            SaveTripBtn.Click();
+            Actions clickToLocation = new Actions(DriverFactory.Driver);
```
Comment (OlegBaslak) [inline Framework/Framework/UI/Pages/CheapFlightPage.cs]: var

File: Framework/Framework/UI/Pages/CheapFlightPage.cs [hunk 4]
```diff
@@ -13,7 +14,7 @@ public class CheapFlightPage
         private const string LowestFareXPath = "//div[@class='calendar-content']//span[(contains(text(), 'Lowest Fare'))]//ancestor::*[position()=2]//p[@class='calendar-price']";
 
         private IWebElement ContinueBtn => DriverFactory.Driver.FindElement(By.XPath("//button[@class='core-btn-primary core-btn-medium action-button__inner']"));
-        private IWebElement SelectRaceBtn => DriverFactory.Driver.FindElement(By.XPath("//button[@id='continue']"));
+        private List<IWebElement> SelectRaceBtn => DriverFactory.Driver.FindElements(By.XPath("//button[@id='continue']")).ToList();
```
Comment (OlegBaslak) [inline Framework/Framework/UI/Pages/CheapFlightPage.cs]: SelectRaceBtn to SelectRaceButtons

Comment (KristinaKosko) [inline Framework/Framework/UI/Pages/CheapFlightPage.cs]: @OlegBaslak,  done

File: Framework/Framework/UI/Pages/CheapFlightPage.cs (hunk 2 above)
Comment (KristinaKosko) [inline Framework/Framework/UI/Pages/CheapFlightPage.cs]: @OlegBaslak , it was resolved in commit "Resolves conflicts"

File: Framework/Framework/UI/Pages/CheapFlightPage.cs [hunk 5]
```diff
@@ -14,7 +14,7 @@ public class CheapFlightPage
         private const string LowestFareXPath = "//div[@class='calendar-content']//span[(contains(text(), 'Lowest Fare'))]//ancestor::*[position()=2]//p[@class='calendar-price']";
 
         private IWebElement ContinueBtn => DriverFactory.Driver.FindElement(By.XPath("//button[@class='core-btn-primary core-btn-medium action-button__inner']"));
-        private List<IWebElement> SelectRaceBtn => DriverFactory.Driver.FindElements(By.XPath("//button[@id='continue']")).ToList();
+        private List<IWebElement> SelectRaceBtns => DriverFactory.Driver.FindElements(By.XPath("//button[@id='continue']")).ToList();
```
Comment (OlegBaslak) [inline Framework/Framework/UI/Pages/CheapFlightPage.cs]: don't reduce names. Should be Buttons

**Pull request closed without merging**